### ISSUE TITLE
Failed to pull image "zookeeper:3.9.0": rpc error: code = NotFound desc = failed to pull and unpack image "docker.io/library/zookeeper:3.9.0": no match for platform in manifest: not found

### ISSUE THREAD

**Issue by SingKS8** [31z4/zookeeper-docker#152]
Tag '3.9.0', now, is missing arch 'linux/amd64', but 'latest' has all arches.

**Comment by 31z4** [comment]
Thanks for reporting! I raised it here https://github.com/docker-library/official-images/pull/15212#issuecomment-1681656477

**Comment by adam-bielasty-yb** [comment]
I am not sure if that's related but

New version does not work for us.

We are getting infos like that:
```
[35mzookeeper_1          |[0m 2023-08-18 08:28:47,579 [myid:] - INFO  [NIOWorkerThread-4:o.a.z.s.NIOServerCnxn@339] - Unable to read additional data from client, it probably closed the socket: address = /172.18.0.4:45754, session = 0x1000001bc950000
```
which then it is causing errors to connect to the nodes.

Nothing changes on our side apart from getting the latest ZooKeeper image **3.9.0**. All worked fine on **3.8.2**.
Do you know what might be the issue?

**Comment by 31z4** [comment]
No @adam-bielasty-yb it's not related to the original issue which was missing architecture for the `3.9.0` tag. The original issue is fixed now so closing this.

Please create a separate bug report including all the relevant issue (e.g., your environment and Zookeeper config).
